server/src/test/java/org/elasticsearch/index/engine/LiveVersionMapTests.java

```diff
@@ -137,24 +142,25 @@ public void testConcurrently() throws IOException, InterruptedException {
                             }
                             if (isDelete == false && rarely()) {
                                 versionValue = new DeleteVersionValue(versionValue.version + 1, versionValue.seqNo + 1,
-                                    versionValue.term, Long.MAX_VALUE);
+                                    versionValue.term, clock.incrementAndGet());
                             } else {
                                 versionValue = new VersionValue(versionValue.version + 1, versionValue.seqNo + 1, versionValue.term);
                             }
                             values.put(bytesRef, versionValue);
                             map.putUnderLock(bytesRef, versionValue);
                         }
+                        if (rarely()) {
+                            map.pruneTombstones(0, 0);
```

Comment:
can you comment we explicitly don't do any pruning? otherwise this looks odd.

Reply:
not sure I understand